### DISCUSSION TITLE
separator filling container issue

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3358] [General] [answered]
hey, I came a cross a small bug with separator component from Paste Components in Figma. The problem is that in some places and situations, the `line` layer inside separator component is not filling container. If the separator had an auto-layout and `line` layer was filled to the container, we would fix this issue, right?

<img width="785" alt="Zrzut ekranu 2023-07-27 o 19 08 21" src="https://github.com/twilio-labs/paste/assets/140410476/f3e8c67e-6d88-4a8d-9bde-e9a8aff8334c">

**Comment by loreina** [comment]
Hi @marektonik,

Thanks for catching this! I'm pushing up a fix now and will update here when that's published in the Paste Components library.

**Comment by loreina** [comment, marked as answer]
Hi @marektonik, a fix was published in the Paste Components library. You should be able to update from the main file and get those changes now. Thanks for reporting this!
